test/benchmark.hs

```diff
@@ -27,18 +35,29 @@
   * emap on PATRICIA tree: ...................
     4.532ns per iteration / 220663.09 per second.
 -}
+{-# LANGUAGE ScopedTypeVariables #-}
+
```

Comment:
Can you please join these three `LANGUAGE` pragmas up (e.g. how it's done in `PatriciaTree`).